frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/create-from-add-options.ts

```diff
@@ -81,10 +81,6 @@ Given('pipeline {string} is executed for 5 times', (pipelineName: string) => {
   cy.get(pipelineRunDetailsPO.pipelineRunStatus).should('not.have.text', 'Running');
 });
 
-When('user enters Git Repo url as {string}', (gitUrl: string) => {
```

Comment:
If the implementation is removed the step from feature file also should be removed right?

Reply:
Its duplicate one, so thought to remove it